flink-learning-common/src/test/java/com/zhisheng/common/utils/DateUtilTests.java

```diff
@@ -0,0 +1,71 @@
+package com.zhisheng.common.utils;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+import static com.zhisheng.common.utils.DateUtil.*;
+import static org.junit.Assert.*;
+
+public class DateUtilTests {
+
+    @Test
+    public void testFormat() throws ParseException {
+        SimpleDateFormat myDate1 = new SimpleDateFormat("yyyy-MM-dd");
+        SimpleDateFormat myDate2 = new SimpleDateFormat("yyyy-MM-dd HH:mm");
+        SimpleDateFormat myDate3 = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        SimpleDateFormat myDate4 = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.0");
```

Comment:
Change names to `simpleDateFormatN` I think, for clarity

Reply:
removed